### ISSUE TITLE
commit titles

### ISSUE THREAD

**Issue by panckreous** [whoisdsmith/BOOKMRKS-MTHRFCKR-II#1]
hey once this is stable and whatnot, what do you think about commit titles and messages that summarize the changes? not sure if thatd be possible, feel free to close this either way

**Comment by whoisdsmith** [comment]
yes, eventually I plan to automate all changes and updates to commit what has changed and what was added!
